memberships/tests/test_stripe_webhooks.py

```diff
@@ -114,3 +116,25 @@ def test_a_successful_payment_for_membership_gets_logged_in_db(self):
         payments = Payment.objects.all()
 
         self.assertEqual(1, payments.count())
+
+    def test_a_member_is_given_a_membership_renewal_date_upon_payment(self):
```

Comment:
I think we should have separate test cases for a new member being given a renewal date and an existing member getting a renewal date.

For two main reasons:
1. Tests can act as a form of documentation, so it's nice to have it explicitly laid out that "new members renew a year from new" and "existing members renew a year from their previous renew date"
2. These are two separate code branches. If I delete the else from the `set_sand_renewal_date` function this test will still pass, but the functionality will be broken.